recipes/hwm/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "hwm" %}
+{% set version = "1.0.1" %}
+{% set python_min = "3.6" %}  
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 5f09159edf703390f4d3110d4737b2f2c47754a8bf11c2b0993e67270773f8c8
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
Please use grayskull to generate Python recipes. Grayskull make it easier for us to review.

See https://github.com/conda-forge/staged-recipes?tab=readme-ov-file#grayskull---recipe-generator-for-python-packages-on-pypi for more information.